src/clj/puppetlabs/services/request_handler/request_handler_service.clj

```diff
@@ -2,14 +2,8 @@
   (:require [puppetlabs.trapperkeeper.core :as tk]
             [puppetlabs.services.protocols.request-handler :as handler]
             [puppetlabs.services.request-handler.request-handler-core :as core]
-            [puppetlabs.services.jruby.jruby-puppet-service :as jruby]
             [puppetlabs.trapperkeeper.services :as tk-services]))
 
-(defn- handle-request
-  [request jruby-service config]
-  (jruby/with-jruby-puppet jruby-puppet jruby-service
-    (core/handle-request request jruby-puppet config)))
-
```

Comment:
Why did you make this change?  I try to keep interactions with services contained within "service" namespaces - which was the origin reason for getting the `jruby-puppet` instance here, instead of passing the JRuby service reference down into `handle-request`.